app/src/util/form-schema/manual-input-hierarchy.json

```diff
@@ -1490,6 +1560,835 @@
+            "waterborne-navigation-vehicle-type": "vehicle-type-all"
+          },
+          {
+            "id": "geographic-waterborne-navigation-activity",
+            "waterborne-navigation-vehicle-type": "vehicle-type-ferries"
+          },
+          {
+            "id": "geographic-waterborne-navigation-activity",
+            "waterborne-navigation-vehicle-type": "vehicle-type-boats"
+          }
+        ]
+      }
+    ],
+    "directMeasure": {
+      "id": "waterborne-navigation-direct-measure-methodology",
+      "extra-fields": [
+        {
+          "id": "waterborne-navigation-fuel-type",
+          "options": [
+            "fuel-type-gasoline",
+            "fuel-type-diesel",
+            "fuel-type-lpg",
+            "fuel-type-cng",
+            "fuel-type-biofuel"
+          ]
+        },
+        {
+          "id": "waterborne-navigation-vehicle-type",
+          "options": [
+            "vehicle-type-ferries",
+            "vehicle-type-boats",
+            "vehicle-type-marine-vessels"
+          ],
+          "multiselect": true
+        },
+        {
+          "id": "waterborne-navigation-fuel-source",
+          "type": "text"
+        }
+      ]
+    }
+  },
+  "II.3.2": {
+    "methodologies": [
+      {
+        "id": "electricity-consumption-waterborne-navigation-methodology",
+        "activities": [
+          {
+            "id": "electricity-consumption-waterborne-navigation-activity",
+            "unique-by": [
+              "waterborne-navigation-vehicle-type",
+              "waterborne-navigation-electricity-source"
+            ],
+            "extra-fields": [
+              {
+                "id": "waterborne-navigation-vehicle-type",
+                "options": [
+                  "vehicle-type-all",
+                  "vehicle-type-ferries",
+                  "vehicle-type-boats",
+                  "vehicle-type-marine-vessels"
+                ]
+              },
+              {
+                "id": "waterborne-navigation-electricity-source",
+                "type": "text"
+              }
+            ]
+          }
+        ],
+        "suggestedActivities": [
+          {
+            "id": "electricity-consumption-waterborne-navigation-activity",
+            "waterborne-navigation-vehicle-type": "vehicle-type-all"
+          },
+          {
+            "id": "electricity-consumption-waterborne-navigation-activity",
+            "waterborne-navigation-vehicle-type": "vehicle-type-ferries"
+          },
+          {
+            "id": "electricity-consumption-waterborne-navigation-activity",
+            "waterborne-navigation-vehicle-type": "vehicle-type-boats"
+          }
+        ]
+      }
+    ],
+    "directMeasure": {
+      "id": "waterborne-navigation-direct-measure-energy-consumption-methodology",
+      "extra-fields": [
+        {
+          "id": "waterborne-navigation-vehicle-type",
+          "options": [
+            "vehicle-type-ferries",
```

Comment:
missing options:
the option: "vehicle-type-all"
"exclusive": "vehicle-type-all"